setup.cfg

```diff
@@ -13,7 +13,7 @@ upload-dir = docs/_build/html
 show-response = 1
 
 [tool:pytest]
-minversion = 3.0
+minversion = 6.0
```

Comment:
Why do we need such a new pytest? even astropy is stuck with an older one

Reply:
I am not confident that older pytest would work flawlessly. But you can bump it down to 5.0 if you wish. Sorry if I bumped it up too high, I thought it might be okay given pytest is at 6.2 now.